### PR TITLE
Added support for strikethrough color

### PR DESCRIPTION
**Pull request by htinlinn**

### CODE REVIEW

File: Marker/Classes/TextStyle.swift [hunk 1]
```diff
@@ -161,6 +168,7 @@ public struct TextStyle {
         self.textAlignment = textAlignment
         self.lineBreakMode = lineBreakMode
         self.strikethroughStyle = strikethroughStyle
+        self.strikethroughColor = strikethroughColor
```
Comment (jgsamudio) [inline Marker/Classes/TextStyle.swift:171]: Can you update the example project with the new color?

**Review by jgsamudio — COMMENTED** [review]
Can we also add a test for color attributes?

File: Marker/Classes/TextStyle.swift (hunk 1 above)
Comment (htinlinn) [reply Marker/Classes/TextStyle.swift:171]: done

**Comment by htinlinn** [comment]
@jgsamudio I think the attributes test should be in a separate PR. So far, there hasn't been any test for other attributes yet.

**Review by jgsamudio — APPROVED** [review]

**Pull request merged**